### ISSUE TITLE
Add spack support

### ISSUE THREAD

**Issue by SamTov** [zincware/MDSuite#220]
**What feature would you like to see added?**
Add a spack install script so that MDSuite can be installed with spack.

**Describe how this feature would improve the code**
It would allow for easier installation on clusters running spack.

**Describe how this feature would work/possible implementations**
Check the [spack](https://github.com/spack/spack) repo and see how to build the install scripts.

**Comment by PythonFZ** [comment]
I think that releasing it on PyPi and for conda should be prioritized?

If we release MDSuite on PyPi everyone can easily install it into their environment?
I would also emphasize to make it a conda package as well which allows also for a very easy installation on a Cluster and we can include GPU support.

If this is done, we could think about making it a spack package (see [Package Creation Tutorial for spack](https://spack-tutorial.readthedocs.io/en/latest/tutorial_packaging.html#packaging-tutorial))?

**Comment by SamTov** [comment]
Yeah I agree that we should prioritise Conda and Pip but that's orthogonal to also considering putting it on spack. We can put the raw package on spack for one and the code is extremely deployable on clusters so I think it is nice to have it in our minds as a priority. It also shouldn't take a very long time and shows that we care for people who don't want to deal with learning how to manage environments and things on remote machines.
